string command
---
bool received_status
